docs/docs/components.mdx

```diff
@@ -955,58 +974,85 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
     "use_shared_vocab": False
   ```
 
+  **Configuring for incremental training**
+
+  This component needs to account for the extra vocabulary that new training examples will
+  bring in during the process of [incremental training](./command-line-interface.mdx#incremental-training).
+  Use `additional_vocabulary_size` while training the base model from scratch to account for this:
+
+  ```yaml-rasa
+  pipeline:
+  - name: CountVectorsFeaturizer
+    additional_vocabulary_size:
+      text: 1000
```

Comment:
I think we should describe what `text`, `response` and `action_text` stand for

Reply:
Added a few lines to the description.